aperturedb/OM/Repository.py

```diff
@@ -0,0 +1,169 @@
+from typing import Dict, List, Optional
+from aperturedb.OM.Cursor import Cursor
+from aperturedb.OM.Entity import Entity
+from aperturedb import Connector
+
+
+class Repository:
+    """**Repository can be thought of as a store of homogenous entities**
+
+    Notes:
+        - Ideally, there should be a way to convert to/from pytorch dataset into Data in aperturedb.
+        - This has options to extract the entities in the database in various ways, controled by filter ctriteria.
+    
+    Example::
+    
+        dataset = Repository.filter(constrints)
+        rectangles, labels = pytorch.infer(dataset)
+        dataset.add_BoundingBoxes(rectangles, labels)
+    """
+
+    def __init__(self, db: Connector) -> None:
+        self._object_type = "Entity"
+        self._db = db
+        self._entity_types = {
+            "Entity": Entity
+        }
+
+    def _find_command(self):
+        return f"Find{self._object_type}"
+
+    def get(self, id) -> Entity:
```

Comment:
we should try to avoid exposing uniqueid to the user, given that the OM is a higher level abstraction.
but the id in this case can be a local id (or the index) of the object within the repository.